.github/mergeable.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+mergeable:
+  - when: schedule.repository
+    validate:
+      - do: stale
+        days: 20
+        type: issues
+    filter:
+      - do: repository
+        name:
+          or:
+            - must_include:
+              regex: 'pi-hole'
+            - must_include:
+              regex: 'adminLTE'
+    pass:
+      - do: comment
+        payload:
+          body: This item is 20 days or more old. Is this item still relevant?
```

Comment:
```suggestion
          body: This issue is 20 days or more old. Is this item still relevant?
```

Reply:
I thought about that, eventually we can add PRs to the stale check so that's why I called it `item`. Since it's currently just targeted to Issues then I agree that we should call it that.